synapse/lib/storm.py

```diff
@@ -2926,3 +2926,62 @@ async def execStormCmd(self, runt, genr):
 
                     async for node in self.iterEdgeNodes(verb, idenset, n2):
                         yield node, _path.fork(node)
+
+class EdgeDelCmd(Cmd):
+    '''
+    Bulk delete light edges from input nodes.
+
+    Examples:
+
+        # Delete all "foo" light edges from an inet:ipv4
+        inet:ipv4=1.2.3.4 | edge.del foo
+
+        # Delete light edges with any verb from a node
+        inet:ipv4=1.2.3.4 | edge.del *
+
+        # Delete all "foo" light edges to an inet:ipv4
+        inet:ipv4=1.2.3.4 | edge.del foo --n2
+    '''
+    name = 'edge.del'
```

Comment:
lets pluralize to `edges.del` ( sorry about the nit picking, i promise to mash approve :D )

Reply:
haha, it's all good, updated to `edges.del`